ADC/Core/Src/main.c

```diff
@@ -0,0 +1,148 @@
+/*
+ * 	Implement a routine that reads the internal voltage reference (ADC_IN17)
+ * 	and internal temperature sensor (ADC_IN18).
+ * 	The internal reference voltage must be in millivolts and the temperature in Celsius.
+ *
+ * 	1 - Polling		[/]
+ * 	2 - Interrupt	[ ]
+ * 	3 - DMA			[ ]
+ */
+
+#include "stm32l053xx.h"
+#include "macros.h"
+#include "init.h"
+#include "struct.h"
+
+ADC_Status	ADC_Config (uint32_t Channel);	//we call this function with the ADC channel we want to read
+void 		Wait (uint16_t);				//delay function, system clock based
```

Comment:
Besides for the Peripheral Init and config functions, all functions must start with lower case. Please write the "w" of Wait function in lower case and remove the space between the parenthesis and the last letter of the functions.
The function name format that we use is: 
```
void one(int x);
void twoWords(int x, int y);
void threeWordsFoo(int x, int y);
void ADC_init();
void SPI_deInit();
void GPIO_config();
```